core/src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -48,10 +48,7 @@ class TiContext(val sparkSession: SparkSession) extends Serializable with Loggin
     } else Option.empty)
   final val tiSession: TiSession = TiSession.getInstance(tiConf)
   lazy val sqlContext: SQLContext = sparkSession.sqlContext
-  lazy val tiConcreteCatalog: TiSessionCatalog =
-    new TiConcreteSessionCatalog(this)(new TiDirectExternalCatalog(this))
   lazy val sessionCatalog: SessionCatalog = sqlContext.sessionState.catalog
```

Comment:
Should delete this?

Reply:
i think it should be deleted. it's useless after we only support catalog plugin mode.

Reply:
It hasn't been deleted yet